### ISSUE TITLE
importantForAccessibility= Yes still has access to child components

### ISSUE THREAD

**Issue by jessamp** [microsoft/react-native-windows#9527]
I'm trying to make a parent component accessible, and the individual child components ignored. I'm setting the View to be accessible and importantForAccessibility=Yes, but the child components are still being highlighted by the narrator.

**Comment by NickGerleman** [comment]
I think this is not yet implemented. See https://github.com/microsoft/react-native-windows/issues/5113

**Comment by AgneLukoseviciute** [comment]
> I think this is not yet implemented. See #5113

`no-hide-descendants` would make the parent inaccessible also. From my understanding, this is behaving as expected. To get the behavior described above we'd need a value akin to "yes-hide-descendants", but that's not an option. 

To get the children ignored by accessibility in this case, I think you'll have to manually set importantForAccessibility=No on them.

**Comment by chrisglein** [comment]
> I'm trying to make a parent component accessible, and the individual child components ignored. I'm setting the View to be accessible and importantForAccessibility=Yes, but the child components are still being highlighted by the narrator.

Are you coming from some cross-platform assumptions here? There are behaviors on other platforms as far as how setting accessibility changes the visibility of children. It'd be helpful to know which platforms you're dealing with outside of Windows.

@AgneLukoseviciute can you compare Windows' behavior here to the other platforms and see how we compare? There should be specs on how the browsers interpret these that should be aligned, if these map to the web properties.
